packages/core/contracts/oracle/implementation/GovernorV2.sol

```diff
@@ -208,7 +230,42 @@ contract GovernorV2 is MultiRole, Lockable {
         return OracleGovernanceInterface(finder.getImplementationAddress(OracleInterfaces.Oracle));
     }
 
+    function _getSafetyModule() private view returns (SafetyModuleInterface) {
+        return SafetyModuleInterface(getMember(uint256(Roles.EmergencyProposer)));
+    }
+
     function _getIdentifierWhitelist() private view returns (IdentifierWhitelistInterface) {
         return IdentifierWhitelistInterface(finder.getImplementationAddress(OracleInterfaces.IdentifierWhitelist));
     }
+
+    function _appendTransactionsToProposal(Transaction[] memory transactions, Proposal storage proposal) internal {
```

Comment:
logic that was internal within the two other functions now pulled to be re-used.